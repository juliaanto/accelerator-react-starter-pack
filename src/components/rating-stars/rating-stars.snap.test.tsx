import RatingStars from './rating-stars';
import {render} from '@testing-library/react';

describe('Component: RatingStars', () => {
  test('should render correctly when rating = 0', () => {
    const {container} = render(<RatingStars rating={0} width={14} height={14} />);
    expect(container).toMatchSnapshot();
  });

  test('should render correctly when rating = 3', () => {
    const {container} = render(<RatingStars rating={3} width={14} height={14} />);
    expect(container).toMatchSnapshot();
  });

  test('should render correctly when rating = 5', () => {
    const {container} = render(<RatingStars rating={5} width={14} height={14} />);
    expect(container).toMatchSnapshot();
  });
});
