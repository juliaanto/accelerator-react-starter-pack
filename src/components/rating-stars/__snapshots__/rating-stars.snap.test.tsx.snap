// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: RatingStars should render correctly when rating = 0 1`] = `
<div>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
</div>
`;

exports[`Component: RatingStars should render correctly when rating = 3 1`] = `
<div>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
</div>
`;

exports[`Component: RatingStars should render correctly when rating = 5 1`] = `
<div>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="14"
    width="14"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
</div>
`;
